pootle/static/js/shared/components/AboutDialog.js

```diff
@@ -31,22 +32,25 @@ const AboutDialogContent = React.createClass({
           <h1>Zing</h1>
 
           <p>
-            This server is powered by <strong>Zing</strong> —
+            {tct(`This server is powered by %(zing)s —
```

Comment:
Given that we use that in some other places, I'm guessing it should work, although I haven't tested that myself. JS strings are extracted into a master resource file by leveraging `xgettext` (run `make pot` for running the script that does everything extraction-related).